app/models/dialog_field_check_box.rb

```diff
@@ -30,7 +30,7 @@ def normalize_automate_values(automate_hash)
   def refresh_json_value
     @value = values_from_automate
 
-    {:checked => checked?}
+    {:checked => checked?, :read_only => read_only}
```

Comment:
Sure, we can do that. It doesn't matter too much since the javascript will only check if it's true or not, so if it's nil it treats it like a false, but it's probably better for readability to have it be `read_only?` as you suggest.